src/App.tsx

```diff
@@ -1,61 +1,174 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  return usersFromServer.find((user) => user.id === userId) || null;
+}
+
+export const todos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+const allowedChars
+    = '0123456789abcdefghigklmnopqrstuvwxyz абвгдеёжзийклмнопрстуфхцчшщъыьэюя';
```

Comment:
why is the Ukrainian language not supported?)

Reply:
It's not my fault) Here is the task:
1. (\* **Optional**) Allow entering only letters (`ru` and `en`), digits, and `spaces` in the `title` field. Just remove any other characters from the `title`.